src/main/kotlin/net/leanix/githubagent/services/GitHubGraphQLService.kt

```diff
@@ -67,6 +68,37 @@ class GitHubGraphQLService(
         }
     }
 
+    fun getFileContent(
+        owner: String,
+        repositoryName: String,
+        filePath: String,
+        token: String
+    ): String {
```

Comment:
If the file path is always the path to the manifest file, then we should change the function name to `getManifestFileContent` and hard code the file path inside the function.